From 79e90baca6442c0fb3627dd1fa1b083c8df4fa01 Mon Sep 17 00:00:00 2001
From: Bettina Cerban <bettinacerban@gmail.com>
Date: Thu, 18 Apr 2019 11:46:11 -0300
Subject: [PATCH] filter config values on testSuiteStart

---
 .../Magento/TestFramework/Isolation/DeploymentConfig.php        | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dev/tests/integration/framework/Magento/TestFramework/Isolation/DeploymentConfig.php b/dev/tests/integration/framework/Magento/TestFramework/Isolation/DeploymentConfig.php
index 3132aed4d21e..ae59bc004db0 100644
--- a/dev/tests/integration/framework/Magento/TestFramework/Isolation/DeploymentConfig.php
+++ b/dev/tests/integration/framework/Magento/TestFramework/Isolation/DeploymentConfig.php
@@ -51,7 +51,7 @@ public function startTestSuite()
     {
         if (null === $this->reader) {
             $this->reader = Bootstrap::getObjectManager()->get(\Magento\Framework\App\DeploymentConfig\Reader::class);
-            $this->config = $this->reader->load();
+            $this->config = $this->filterIgnoredConfigValues($this->reader->load());
         }
     }
 
