diff --git a/phpserver/router.php b/phpserver/router.php
index 06c23ecd..628193f5 100644
--- a/phpserver/router.php
+++ b/phpserver/router.php
@@ -14,6 +14,15 @@
  * example usage: php -S 127.0.0.41:8082 -t ./pub/ ./router.php
  */
 
+header('Access-Control-Allow-Origin: *');
+header('Access-Control-Allow-Methods: *');
+header('Access-Control-Allow-Headers: DNT,X-CustomHeader,Keep-Alive,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,Content-Range,Range,Authorization');
+
+if ($_SERVER['REQUEST_METHOD'] == 'OPTIONS') {
+    http_response_code(200);
+    exit;
+}
+
 /**
  * Set it to true to enable debug mode
  */
